lnd_test.go

```diff
@@ -234,7 +244,7 @@ func closeChannelAndAssert(ctx context.Context, t *harnessTest, net *networkHarn
 // numOpenChannelsPending sends an RPC request to a node to get a count of the
 // node's channels that are currently in a pending state (with a broadcast, but
 // not confirmed funding transaction).
-func numOpenChannelsPending(ctxt context.Context, node *lightningNode) (int, error) {
+func numOpenChannelsPending(ctxt context.Context, node *lntest.HarnessNode) (int, error) {
```

Comment:
This commit is out of order FWIW (it wouldn't compile by itself).

Reply:
I think GitHub just orders the commits weird (by date instead of actually in order). The dates are screwed up because I had to cherry-pick a bunch from another branch.